src/box3d.rs

```diff
@@ -176,7 +192,7 @@ where
     /// nonempty but this box3d is empty.
     #[inline]
     pub fn contains_box(&self, other: &Self) -> bool {
-        other.is_empty()
+        other.is_empty_or_negative()
```

Comment:
uh, this can be super tricky
as a user, I'd assume that if A contains B, then B min/max is fully within A min/max
I know that this was already broken by special handling of empty boxes, but now it's getting even more out of hand

Reply:
I don't like this behavior either (I'd prefer to not special case empty rects nor negative) but since it's how `TypedRect` work I think that the safest route is to do the same.

In practice empty and negative boxes have the same meaning so I think that the least confusing is  to treat them the same way in `contains_box`. Box2D does that as well.